app/src/main/res/values-am-rET/strings.xml

```diff
@@ -580,8 +580,6 @@
   <string name="assigned_facility_view_title">የተመደቡበት ተቋም</string>
   <string name="assigned_facility_view_change">ለውጥ</string>
 
-  <!-- Facility Picker -->
-  <string name="facilitypicker_search_hint">ክሊኒክዎን ይፈልጉ</string>
```

Comment:
We don't need to remove this. Since its going to be the same key. The transifex will update the translations.